Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/DriverOnboarding.yaml

```diff
@@ -458,6 +458,46 @@ VehiclePUC:
       where:
         and: [rcId, driverId]
 
+FleetRCAssociation:
+  tableName: fleet_rc_association
+  fields:
+    id: Id DriverRCAssociation
+    driverId: Maybe (Id Person)
+    fleetOwnerId: Id Person
+    rcId: Id VehicleRegistrationCertificate
+    associatedOn: UTCTime
+    associatedTill: Maybe UTCTime
+    rcVerificationStatus: VerificationStatus
```

Comment:
Why rcVerificationStatus and vehicleRegistrationNo in this table?

Reply:
so for this we have to show the status of each rc , so instead of fetching from rc status from each table we get it from here only. therefor stored vehicle no also in this only.

Reply:
No bro, it should not be done like that. This table will grow as association will keep changing. We should not keep replicating the all data. Lets remove it from here.